onadata/libs/utils/logger_tools.py

```diff
@@ -239,7 +247,7 @@ def create_instance(username, xml_file, media_files,
                 #    has already been submitted for that user.
                 raise DuplicateInstance()
 
-        # get new and depracated uuid's
+        # get new and deprecated uuid's
         new_uuid = get_uuid_from_xml(xml)
         duplicate_instances = Instance.objects.filter(uuid=new_uuid)
```

Comment:
Somehow `uuid` has a Postgres index (on both HHI and OCHA dbs). That's great, but I'd love to know why, since it's neither unique nor specifies `db_index`: https://github.com/kobotoolbox/kobocat/blob/ea6d9831a973a04a117ae9a3b79cbb35d587cb53/onadata/apps/logger/models/instance.py#L130.

Reply:
Was it added manually, I wonder, to these production databases? I don't see the index on my local database